go.mod

```diff
@@ -2,9 +2,24 @@ module github.com/19chonm/461_1_23
 
 go 1.19
 
-require github.com/spf13/cobra v1.6.1
-
 require (
+	github.com/gofiber/fiber/v2 v2.41.0
```

Comment:
Are there any unnecessary imports in the go.mod and go.sum? No idea if these are correct or not, I just wanted to make sure you double-checked.

Reply:
I don't think so? go errors out for me if there are.